src/helpers/lists.js

```diff
@@ -4,7 +4,8 @@ export const languages = {
   rb: 'ruby',
   ru: 'ruby',
   md: 'markdown',
-  py: 'python'
+  py: 'python',
+  yml: 'yaml'
```

Comment:
@tbrandle let's also add `yaml: 'yaml'` since both `.yml` and `.yaml` are valid yaml extensions

Reply:
@5290charlie  It should work with both because of the logic in TextEditor
```
  const { languages } = lists
  const lang = languages[ext] || ext
```